iam_mapping.py

```diff
@@ -40,17 +45,38 @@ def delete_mapping(body: dict, meta: dict, spec: dict, **kwargs):
     updated_mapping = delete_user(spec, users)
     apply_mapping(cm, updated_mapping)
 
+
+def deploy_crd_definition():
+    with open("kubernetes/iamidentitymappings.yaml", 'r') as stream:
+        try:
+            body = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            logger.error(exc)
+            exit(1)
```

Comment:
Instead of exiting you can `raise` without `exit(1)`. On the plus side you will have a stack when your program exits that way.